packages/@aws-cdk/cfnspec/spec-source/000_sam.spec.json

```diff
@@ -375,6 +392,50 @@
     "ResourceSpecificationTransform": "AWS::Serverless-2016-10-31",
     "ResourceSpecificationVersion": "2016-10-31",
     "ResourceTypes": {
+        "AWS::Serverless::Application": {
+            "Documentation": "https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessapplication",
+            "Properties": {
```

Comment:
Do we have an official spec to define these properties? @jlhood

Reply:
The documentation you linked is the official spec:

https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessapplication

We don't have an official machine-readable spec for SAM. My understanding is CDK has previously depended on [a schema for SAM from the goformation project](https://github.com/awslabs/goformation/blob/master/schema/sam.schema.json), which is not maintained by the SAM team.

Reply:
Copy @PaulMaddox

Reply:
Update as a patch